android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerViewManager.java

```diff
@@ -124,6 +127,18 @@
 
     public ReactTVExoplayerViewManager(ReactApplicationContext reactApplicationContext) {
         this.reactApplicationContext = reactApplicationContext;
+        setDebug();
+    }
+
+    private void setDebug() {
+        final boolean debug = false;
```

Comment:
It's usually the convention to write final variables in uppercase. Also booleans usually start with `is`, `has`, etc

```suggestion
        final boolean IS_DEBUG = false;
```